src/staticanalysis/bot/tests/test_clang.py

```diff
@@ -105,7 +105,8 @@ def test_as_markdown(mock_revision):
         'code': 'clang-tidy.dummy-check',
         'line': 42,
         'name': 'Clang-Tidy - dummy-check',
-        'description': 'dummy message\nChecker reliability is high (false positive risk).\n\n > Dummy body',
+        'description': 'dummy message\nChecker reliability is high, this mean that the risk to'
```

Comment:
@sylvestre Do you approve this phrasing ?

Reply:
I am not a huge fan, I think it is too long for stuff that we publish a lot...
I would prefer something like:
`Checker reliability is high, meaning that the false positive ratio is low.`